README.md

```diff
@@ -111,19 +111,29 @@ All the data (past and coming) are available publicly in JSON:
+* 01-02: [DevCon 2023 Romania](https://www.dev-con.ro/) - Bucharest (Romania)
+* 01-03: [Ektoparty 2023](https://ekoparty.org/) - Buenos Ares (Argentina)
 * 2: [GopherCon Ireland](https://gophercon.ie/) - Dublin (Ireland) <a href="https://www.papercall.io/gopherconie"><img alt="GopherCon Ireland 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-July-2023&color=red"></a>
+* 02: [Data.TLV Summit 2023](http://datatlv.com/) - Tel Aviv (Israel)
+* 02: [Day of Shecurity](https://www.dayofshecurity.com/nov-2023-conference) - VIRTUAL ()
 * 2-3: [Agile Tour Nantes](https://agilenantes.org/evenements/agile-tour-nantes/agile-tour-nantais-2023/) - Nantes (France)
 * 2-3: [GopherCon Singapore and Southeast Asia](https://2023.gophercon.sg/) - Singapore (Singapore)
 * 3: [XCraft](https://xcraft.fr) - Lyon (France)
 * 3: [DevFest Uyo](https://gdg.community.dev/events/details/google-gdg-uyo-presents-devfest-uyo-2023/) - Uyo (Nigeria)
 * 3: [SyliusCon](https://sylius.com/conference) - Poznan (Poland)
 * 3: [City JS Berlin](https://berlin.cityjsconf.org/home) - Berlin (Germany)
 * 3: [Fyne Conf](https://conf.fyne.io/) - Edinburgh (UK)
+* 03: [Fyne Conf](https://conf.fyne.io/) - Edinburgh (Scotland)
 * 3-4: [DevFest Catania](https://gdg.community.dev/events/details/google-gdg-catania-presents-devfest-catania-2023/) - Catania (Italy)
 * 3-5: [Hackaday Supercon 2023](https://hackaday.com/2023/05/10/supercon-2023-is-on-we-want-you/) - Pasadena (USA) <a href="https://docs.google.com/forms/d/e/1FAIpQLSfYDwIzWTHZ0_7d8GUznm3Z9w3y8aDcV1MVGSUyY1nTcdJ9Jw/viewform?usp=sf_link"><img alt="CFP Hackaday SuperCon 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2018-July-2023&color=red"></a>
 * 4: [DevFest Nairobi](https://gdg.community.dev/events/details/google-gdg-nairobi-presents-gdg-devfest-nairobi-2023/) - Nairobi (Kenya)
 * 4: [DevFest Jalingo 2023](https://gdg.community.dev/events/details/google-gdg-jalingo-presents-devfest-2023-jalingo/) - Jalingo (Nigeria) <a href="https://sessionize.com/devfest-2023-jalingo/"><img alt="CFP DevFest Jalingo 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2020-October-2023&color=red"></a>
 * 4: [DevFest Ado-Ekiti](https://gdg.community.dev/events/details/google-gdg-ado-ekiti-presents-devfest-ado-ekiti-2023/) - Ado (Nigeria) <a href="https://sessionize.com/devfest-ado-ekiti-2023/"><img alt="CFP DevFest Ado" src="https://img.shields.io/static/v1?label=CFP&message=until%2016-October-2023&color=red"></a>
+* 04: [DevFest Jailingo](https://gdg.community.dev/events/details/google-gdg-jalingo-presents-devfest-2023-jalingo/) - Jailingo (Nigeria)
+* 04: [TedxGrandJunction](http://tedxgj.com/) - Grand Junction, CO (USA)
```

Comment:
```suggestion
* 04: [TedxGrandJunction](http://tedxgj.com/) - Grand Junction (USA)
```

Reply:
"Grand Junction" could refer to a city in Colorado,  Idaho, Iowa, Michigan, or Tennessee

Likewise (and more commonly) "Kansas City" could be in Kansas or Missouri. So I believe specifying the state can be helpful. But I'm not sure how you'd like to handle it overall.

Reply:
I understand, thanks for the explaination :)